app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -4,7 +4,10 @@ module MiddlewareServerHelper::TextualSummary
   #
 
   def textual_group_properties
-    %i(name)
+
+    @prop_hash = eval(@record.properties)
```

Comment:
On 14 Dec 2015, at 10:26, abonas wrote:

>> @@ -4,7 +4,10 @@ module MiddlewareServerHelper::TextualSummary
>>  #
>>
>>  def textual_group_properties
>> -    %i(name)
>> +
>> +    @prop_hash = eval(@record.properties)
>
> what is the purpose of the prop_hash?

Properties are returned as a string from the DB,
so I needed to turn them into a hash here to 
be able to access the individual items like
version or product.